### ISSUE TITLE
Quirky web interface 

### ISSUE THREAD

**Issue by E3V3A** [mundanelunacy/covidmaps#15]
Hi @mundanelunacy  
Great idea, and I've been looking for something like this.
- Easy to use, everybody can enter data.

However, I tried to enter a few items, and noticed the following UI/UX issues:
- [ ] Map starts centered where (where you are) and not near where I am (in Europe). Perhaps you can implement a simple but very rudimentary IP check?
- [ ] Zooming out fully to get a whole picture of Europe (for example) is blocked at a certain zoom level and seriously slowing down browser. I suspect you did this on purpose to not crash the browser when getting millions of points on one map. (Only after, I discovered the search function actually works great.) - Suggestion is not to render any "dots" after that level. 
- [ ] Navigating to "Submit case" is confusing:  
  (a)  as you say "step-1" and "step-2" when it seem that you do not need both, but **one** of them.  
  (b) I tried to enter a few (real) points manually, but they never showed up on the map.
- [ ] The google location data upload button is looking broken with text outside (but still working)
- [ ] There is no back button to get you back to the map once you've entered the new data.

**Comment by mundanelunacy** [comment]
@E3V3A 

Thanks for the feedback.  I work on making this project easier for developers such as yourself to fork 

> Map starts centered where (where you are) and not near where I am (in Europe). Perhaps you can implement a simple but very rudimentary IP check?

Sure.  Still in development so simply I've just centering where there is data

>  Zooming out fully to get a whole picture of Europe (for example) is blocked at a certain zoom level and seriously slowing down browser. I suspect you did this on purpose to not crash the browser when getting millions of points on one map. (Only after, I discovered the search function actually works great.) -

correct.  The bigger the map, the bigger the server payload.  

>  Suggestion is not to render any "dots" after that level.

Don't you think that's confusing to users ?

>  Navigating to "Submit case" is confusing:
> (a) as you say "step-1" and "step-2" when it seem that you do not need both, but one of them.

Yeah, that copy needs fixing.  Any interest to take a crack at it ?  

> (b) I tried to enter a few (real) points manually, but they never showed up on the map.
>  The google location data upload button is looking broken with text outside (but still working)

Any chance you could provide details on how to reproduce the bug?

>  There is no back button to get you back to the map once you've entered the new data.

Easy enough

**Comment by E3V3A** [comment]
> Any chance you could provide details on how to reproduce the bug?

Yes, open firefox and open the map, and try to enter something manually.

**Comment by mundanelunacy** [comment]
ah, I see.  All entries from that "submit" page were added to the database as "unverified" data.  In my previous pull, I set the default config to filter these markers out.  Updated the site, so try searching for the locations you entered again

**Comment by E3V3A** [comment]
@mundanelunacy 
That worked. 

Also, clicking on any of the menu items: *about, privacy* or *donate*. Does nothing.

Now another thing. 
What is the usage scenario you imagine?

**Comment by mundanelunacy** [comment]
Yeah.  There's a lot more I need to polish up.  

I'm looking to merge this project with the efforts of healthcare authorities and local governments.  

Goal is to replicate the Korea model where a professional contract tracer interviews patients and inputs only "verified" data

**Comment by E3V3A** [comment]
> Goal is to replicate the Korea model where a professional contract tracer interviews patients and inputs only "verified" data

Aha, ok, but I'm not sure "contact tracer" is an official profession yet... :eyes: 

However, this project may be useful in similar scenarios. As an example. 
* User takes a test and get a result according to some scheme (I mentioned in other issue.) The user also gets a verification code along with the test result. They input this as a way to verify that what they entered is true/valid. There's now a dot on the map. 
* The user has taken an antibody home-test, he finds he's positive and would like to enter his data. Well, now things get interesting. Does he enter his home? Some random place or what?

:up: Clearly in the first case, they data is just the health institution (PoS) they're taking the PCR test at. In the second, not so clear. Nobody should have to enter their home address. Entering a random place defeats the purpose. So what to do? This s why contact tracing maps are more important. You get a whole sequence of paths someone has taken or at least points or interest (POIs), such as bars and restaurants, and then other people can see if they were there at the time. In this case, your map would need to allow the entry of possibly hundreds of points for each person. Which is probably not what you intended (as you said). So in that regard, how are point selected in the Korean map?
